MetaMask/metamask-mobile
# feat: add references and utils for delete storage key

## **Description**

This PR implements a feature for our users get access to delete their Notification's storage keys if need.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to Settings
2. Choose Notifications
3. Make sure its enabled
4. Click on Reset Notifications
5. Proceed 
6. Check if Notifications remain enabled and a toast is shown.

## **Screenshots/Recordings**



### **Before**



### **After**

https://github.com/user-attachments/assets/07ed0870-a6f2-442c-a692-880037f03f8d




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
